lib/Client.js

```diff
@@ -2233,6 +2233,34 @@ class Client extends EventEmitter {
         return this.requestHandler.request("DELETE", Endpoints.CHANNEL_PIN(channelID, messageID), true);
     }
 
+    /**
+    * Delete a webhook message
+    * @arg {String} webhookID
+    * @arg {String} token
+    * @arg {String} messageID
+    * @returns {Promise}
+    */
+    webhookDeleteMessage(webhookID, token, messageID) {
```

Comment:
`*WebhookMessage` would make more sense right? At least grammatically. This does seem weird though since `executeWebhook` wouldn't be consistent in naming.

Reply:
Yes but then you get the case in message where the method is called `deleteWebhook` which makes it sound like you're deleting a webhook that belongs to that message (but then this point will be invalidated if `webhookDelete` merges into `delete` method)